src/sbt-test/interplay/plugin-with-root/test

```diff
@@ -18,8 +18,6 @@ $ exec git branch -u origin/master
 $ exists mock-sbt-plugin/target/scripted-ran
 
 # Make sure publishSigned ran on every project with the right publish settings
-> contains target/scala-2.12/publish-version no-publish:1.2.3
-$ exists target/scala-2.12/publish-version
```

Comment:
I think having these are _really_ useful. (as either `> contains` or `-> contains`).  Losing these throws those assurances away.  In my previous PRs to interplay they were a lifesaver.